zephyr/samples/modules/mcuboot/mcuboot_external_config/keys.c

```diff
@@ -0,0 +1,209 @@
+/*
```

Comment:
This is not a file you should add.

It already included by the application and can be considered a skeleton file.
When building MCUboot with Zephyr the key is specified using `CONFIG_BOOT_SIGNATURE_KEY_FILE` and extracted into `autogen-pubkey.c` which is included in the build.
Ref:
https://github.com/zephyrproject-rtos/mcuboot/blob/cfec947e0f8be686d02c73104a3b1ad0b5dcf1e6/boot/zephyr/CMakeLists.txt#L295-L307